README.md

```diff
@@ -0,0 +1,39 @@
+# Icinga DB
+
+1. [About](#about)
+2. [Documentation](#documentation)
+3. [Installation](#installation)
+4. [Configuration](#configuration)
```

Comment:
Please add absolute anchors including the filename here. This avoids possible failures.

Example:
```
[About](README.md#about)
```

Reply:
For me this opens a new tab instead of jumping in the file - not sure if that is the intended effect?

If it is, we need to change that in the icingaweb2 README as well.